<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="CalculationParameters" name="calculation_parameters" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3_Substances">
       <attribute name="title">
        <string>Substances</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="SubstancesTableView" name="view_substances" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4_Thermodynamic_functions">
       <attribute name="title">
        <string>Thermodynamic functions</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="PlotTFView" name="plot_tf_view" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5_Amounts">
       <attribute name="title">
        <string>Amounts</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="Table" name="amounts_view"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_result_table">
       <attribute name="title">
        <string>Result Table</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="Table" name="result_table_view"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6_Result">
       <attribute name="title">
        <string>Result</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ResultView" name="result_view" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1138</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CalculationParameters</class>
   <extends>QWidget</extends>
   <header>calculationparameters.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SubstancesTableView</class>
   <extends>QWidget</extends>
   <header>substancestableview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotTFView</class>
   <extends>QWidget</extends>
   <header>plottfview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Table</class>
   <extends>QTableView</extends>
   <header>table.h</header>
  </customwidget>
  <customwidget>
   <class>ResultView</class>
   <extends>QWidget</extends>
   <header>resultview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
